components/todo-input.js

```diff
@@ -1,20 +1,96 @@
-import React from "react";
-import styled from "styled-components";
+import React from 'react';
+import styled from 'styled-components';
 
-export default function TodoInput() {
-  return (
-    <Wrapper>
-      리액트-투두
-      <br />
-      시간을 입력하는 칸
-      <br />
-      Todo를 입력하는 칸
-    </Wrapper>
-  );
+export default function TodoInput(props) {
+	const { date, todo, onChangeTodo, onChangeDate, onSubmit } = props;
+
+	const maxLengthCheck = (object) => {
+		if (object.value.length > object.maxLength) {
+			object.value = object.value.slice(0, object.maxLength);
+		}
+	};
+
+	return (
+		<Form>
+			<InputSection>
+				<P>시간</P>
+				<Input
+					maxlength="8"
+					oninput="maxLengthCheck(this)"
+					type="number"
+					placeholder="날짜를 입력하세요 (ex.20200404)"
+					value={date}
+					onChange={onChangeDate}
+				/>
+			</InputSection>
+
+			<InputSection>
+				<P>TODO</P>
+				<TextArea
+					placeholder="할 일을 입력하세요"
+					value={todo}
+					onChange={onChangeTodo}
+				/>
+			</InputSection>
+			<Button onClick={onSubmit}>등록</Button>
+		</Form>
+	);
 }
 
-const Wrapper = styled.div`
-  border: solid 1px;
-  font-size: 18px;
-  flex: 1;
+const Input = styled.input`
+	width: 80%;
+	border-width: 1px;
+	border-style: solid;
+	border-color: rgb(97, 97, 97);
+	border-image: initial;
+	padding: 0.5rem 0.8rem;
+`;
+const TextArea = styled.textarea`
+	width: 80%;
+	height: 30vh;
+	resize: none;
+	border-width: 1px;
+	border-style: solid;
+	border-color: rgb(97, 97, 97);
+	padding: 0.5rem 0.8rem;
+`;
+const P = styled.p`
```

Comment:
```suggestion
const Label = styled.label`
```
```P```라는 이름보다는 이 태그가 input의 label 역할을 하기 때문에(input 에서 어떤 값을 받는지 알려주기)
label 태그 이용하고 ```Label```이라고 이름 붙여주는 게 더 좋을 것 같아요 :) [참고](https://www.codingfactory.net/11008)